Integrations/integration-Intezer.yml

```diff
@@ -0,0 +1,221 @@
+description: Malware detection and analysis based on code reuse
+detaileddescription: |
+  Intezer’s cloud-based malware detection and analysis provides a fast, in-depth understanding of any file by mapping its code DNA.
+  See [https://analyze.intezer.com](https://analyze.intezer.com) for creating API key and more details.
+configuration:
+- display: API Key
+  name: APIKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: useproxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |-
+    var SERVER_API = 'https://analyze.intezer.com/api/';
+    var IS_AVAILABLE_URL = 'is-available';
+    var HASH_URL = 'v1-2/analyze-by-sha256';
+    var ANALYSIS_RESULT_URL = 'v1-2/analyses/';
+    var SHA_256_REGEXP = new RegExp('[A-Fa-f0-9]{64}');
+    var SUCCESS_STATUS = 'succeeded';
+
+    var sendRequest = function(method, url, parameters) {
+        // handle '/' at the end of the url
+        if (url[url.length - 1] === '/') {
+            url = url.substring(0, url.length - 1);
+        }
+        if (!parameters) {
+            parameters = {};
+        }
+        parameters.api_key = params.APIKey;
+        // no need to handle url query encoding in GET (as no such api in Intezer)
+        var result = http(
+            SERVER_API + url,
+            {
+                Headers: {'Content-Type': ['application/json'], 'Accept': ['application/json']},
+                Method: method,
+                Body: method == 'POST' ? JSON.stringify(parameters) : ''
+            },
+            false,
+            params.useproxy
+        );
+
+        // validate response
+        if (result.StatusCode < 200 || result.StatusCode > 299) {
+            switch (result.StatusCode) {
```

Comment:
In all errors - if we get additional details from the service - let's also print them. Especially in the 5xx codes and the default...

Reply:
would also add service name to errors, so it is clear (sometimes when you do `!file` it's hard to know which service failed.

Reply:
![image](https://user-images.githubusercontent.com/15967181/38040018-3ff4000c-32b7-11e8-9213-dcae802a57d1.png)